pkg/installer/deployresources_resources.go

```diff
@@ -13,25 +13,30 @@ import (
 	"github.com/openshift/installer/pkg/asset/ignition/machine"
 	"github.com/openshift/installer/pkg/asset/installconfig"
 
+	"github.com/Azure/ARO-RP/pkg/api"
 	"github.com/Azure/ARO-RP/pkg/util/arm"
 	"github.com/Azure/ARO-RP/pkg/util/azureclient"
 )
 
 func (m *manager) networkBootstrapNIC(installConfig *installconfig.InstallConfig) *arm.Resource {
```

Comment:
Since we don't require a load balancer anymore, we can just gate on if we're using UserDefinedRouting, then don't add the ip configuration to point to the existing load balancer.

Reply:
I think we are already doing that